pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "mt2"
-version = "1.2.3"
+version = "1.3.0"
```

Comment:
This is a breaking change, but one that I hope affects relatively few users.

Reply:
If we're being strict at sem-ver I suppose this should be `2.0.0`. I'm not too worried about running out of positive integers, so maybe we should go for that?

Reply:
But what if we're being strict at [EffVer](https://jacobtomlinson.dev/effver/)?

I don't mind either way.

Reply:
I feel non-strongly :)

Reply:
It is arguably not a major version change even by semver.

We previously documented an assumption that masses are non negative, making the behaviour for negative masses was not defined. This therefore defines new features in a backward compatible way.

The behaviour on all zero masses was a clear bug, fixed here.

The remaining change is to not return negative on invalid or diverging solutions. Also arguably a bug fix? Negative certainly bugged users ;)